subtractFlats.py

```diff
@@ -115,80 +104,93 @@ def main():
     suffix       = '_f2_R22_S11_E000.fits.gz'
 
     # Process Files
-    magnitude = ['18', '15', '14', '13', '12', '10']
+    magnitude = [18, 15, 14, 13, 12, 10]
     extraId   = ['0', '1', '2', '3', '4']
 
```

Comment:
OK that makes sense. Thanks.  

In principle you are right that it is a label.  It tells you which  PhoSim config you used and is set with the extraId card in the command file.  But, the way phosim is implemented this should be a number and it also shows up in the file name. That is then used to grab the file name.  With this info, you can actually go back and look at the fits headers for the output file and see what real configs you used for the particular run.

I should double check, but I remember when I tried to put a "01" or something else in there it was showed up in the file name as "1" so at some point PhoSim processes it as an integer.